### PR TITLE
get image before send to analysis

### LINKED ISSUE #69: Implement precheck against analysis status
**Describe the bug**
If a deployment is scaled out for a large number of pods, sending images to scan will take a long time.

**Steps to reproduce the issue:**
In a test deployment set replicas to 100

**Expected behavior**
Shorten the response time of the anchore-engine using a precheck.

**Additional context**
If an image is sent to analysis, the anchore-engine puts it in the queue and it will affect response time. Using the precheck the queue will be skipped.

### PR DESCRIPTION
**Pull request by pbalogh-sa**
Signed-off-by: Peter Balogh <p.balogh.sa@gmail.com>

| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    |yes
| API breaks?     | no
| Deprecations?   | no
| Related tickets | fixes #69 
| License         | Apache 2.0

### What's in this PR?
Implement precheck

### CODE REVIEW

**Review by bonifaido — APPROVED** [review]
👍

File: pkg/anchore/client.go [hunk 1]
```diff
@@ -93,7 +93,15 @@ func anchoreRequest(path string, bodyParams map[string]string, method string) ([
 }
 
 func getStatus(digest string, tag string) bool {
-	path := fmt.Sprintf("/v1/images/%s/check?history=false&detail=false&tag=%s", digest, tag)
+	params := url.Values{}
+	params.Add("history", "false")
+	params.Add("datall", "false")
```
Comment (orymate) [inline pkg/anchore/client.go:98]: typo?

Comment (pbalogh-sa) [reply pkg/anchore/client.go:98]: thanks

Comment (pbalogh-sa) [reply pkg/anchore/client.go:98]: fixed on master

**Pull request merged**